tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -778,6 +778,23 @@ function containerd_snapshotter_version_check() {
 	fi
 }
 
+function containerd_erofs_snapshotter_version_check() {
+	local container_runtime_version=$(kubectl get node $NODE_NAME -o jsonpath='{.status.nodeInfo.containerRuntimeVersion}')
+	local containerd_prefix="containerd://"
+	local containerd_version=${container_runtime_version#$containerd_prefix}
+	local min_version_major="2"
+	local min_version_minor="2"
+
+	# Extract major.minor (strip patch and prerelease stuff)
+	local major=${containerd_version%%.*}
+	local rest=${containerd_version#*.}
+	local minor=${rest%%[^0-9]*}
+
+	if [ "${min_version_major}" -gt "${major}" ] || { [ "${min_version_major}" -eq "${major}" ] && [ "${min_version_minor}" -ge "${minor}" ]; }; then
```

Comment:
Should this be `[ "${min_version_minor}" -gt "${minor}" ]` otherwise 2.2.x would be rejected?

Reply:
Nice catch!